packages/calypso-e2e/src/jest-conditionals.ts

```diff
@@ -1,5 +1,9 @@
-import { it } from '@jest/globals';
+import { it, describe } from '@jest/globals';
+
+// These allow a conditional skip based on the parameter input.
 
-// This allows a conditional skip based on the parameter input.
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export const itif = ( conditional: boolean ) => ( conditional ? it : it.skip );
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const describeif = ( conditional: boolean ) => ( conditional ? describe : describe.skip );
```

Comment:
I feel like we should still rename this to something like `skipIf` and `skipDescribeIf` so the skip condition is more clear. What do you think about making the change in this PR?

Reply:
Done in [a4ae0cd](https://github.com/Automattic/wp-calypso/pull/57862/commits/a4ae0cde1b998a13f120ce80d35276658bb7f5d6)